### ISSUE TITLE
config test is not able to handle alternative app pathes

### ISSUE THREAD

**Issue by cguentherTUChemnitz** [strukturag/nextcloud-spreedme#50]
I am using the [nextcloud container](https://hub.docker.com/r/wonderfall/nextcloud/). To get persistent and volume mountable app folder i use the nextcloud config entry:

```

  "apps_paths" => array (
      0 => array (
              "path"     => "/nextcloud/apps",
              "url"      => "/apps",
              "writable" => false,
      ),
      1 => array (
              "path"     => "/apps2",
              "url"      => "/apps2",
              "writable" => true,
      ),
  ),
```

The configuration tests in

> https://myNextCloudURL/index.php/apps/spreedme/admin/debug

shows:

> Ran test #2 (testOwncloudJavascriptConfigFile):
>    Error: Did not find OwnCloudConfig.js at apps/spreedme/extra/static/config/OwnCloudConfig.js

The path is wrong, because the app is located in apps2/spreedme...

Nevertheless, the spreedme app runs fine, only the test fails.

**Comment by leonklingele** [comment]
Thanks, it looks like the path to that file was hard-coded to use the apps/ directory.
Fixed in 30e493791f824c5c8d9e1acbab170263cc7664e4

**Comment by cguentherTUChemnitz** [comment]
That looks quite reasonable. Thanks for your fast response. Is it right to hold the ticket open, until the changes are verified and pushed to master ?
